packages/contracts-bedrock/scripts/DeployOPChain.s.sol

```diff
@@ -387,6 +387,10 @@ contract DeployOPChainOutput is BaseDeployIO {
         require(address(game.weth()) == address(delayedWETHPermissionedGameProxy()), "DPG-40");
         require(address(game.anchorStateRegistry()) == address(anchorStateRegistryProxy()), "DPG-50");
         require(game.l2ChainId() == _doi.l2ChainId(), "DPG-60");
+        require(game.l2BlockNumber() == 0, "DPG-70");
+        require(Duration.unwrap(game.clockExtension()) == 10800, "DPG-80");
```

Comment:
Should we add the `maxClockDuration()` check also?

Reply:
Good call: https://github.com/ethereum-optimism/optimism/pull/12355/commits/b8626321b210e7729b137a1633e14525ca3a5374